ush/set_up_verif_global.sh

```diff
@@ -190,6 +230,28 @@ elif [ $machine = "HERA" ]; then
     export obdata_dir="/scratch1/NCEPDEV/global/Mallory.Row/obdata"
     export ccpa_24hr_arch_dir="/scratch1/NCEPDEV/global/Mallory.Row/obdata/ccpa_accum24hr"
     export METviewer_AWS_scripts_dir="/scratch1/NCEPDEV/global/Mallory.Row/VRFY/METviewer_AWS"
+elif [ $machine = "GAEAC5" ]; then
+    export NWROOT="/gpfs/f5/ufs-ard/world-shared/global/glopara/data/nwpara"
+    export HOMEDIR="/gpfs/f5/epic/scratch/$USER"
+    export STMP="/gpfs/f5/epic/scratch/$USER"
+    export PTMP="/gpfs/f5/epic/scratch/$USER"
+    export NOSCRUB="/gpfs/f5/epic/scratch/$USER"
+    export global_archive="/gpfs/f5/ufs-ard/world-shared/global/metplus.data/archive"
+    export prepbufr_arch_dir="/gpfs/f5/ufs-ard/world-shared/global/metplus.data/prepbufr"
+    export obdata_dir="/gpfs/f5/ufs-ard/world-shared/global/metplus.data/obdata"
+    export ccpa_24hr_arch_dir="/gpfs/f5/ufs-ard/world-shared/global/metplus.data/obdata/ccpa_accum24hr"
+    export METviewer_AWS_scripts_dir="/gpfs/f5/ufs-ard/world-shared/global/metplus.data/VRFY/METviewer_AWS"
+elif [ $machine = "GAEAC6" ]; then
+    export NWROOT="/gpfs/f6/bil-fire8/world-shared/global/glopara/data/nwpara"
+    export HOMEDIR="/gpfs/f6/bil-fire8/scratch/$USER"
+    export STMP="/gpfs/f6/bil-fire8/scratch/$USER"
+    export PTMP="/gpfs/f6/bil-fire8/scratch/$USER"
+    export NOSCRUB="/gpfs/f6/bil-fire8/scratch/$USER"
+    export global_archive="/gpfs/f6/bil-fire8/world-shared/global/metplus.data/archive"
+    export prepbufr_arch_dir="/gpfs/f6/bil-fire8/world-shared/global/metplus.data/prepbufr"
+    export obdata_dir="/gpfs/f6/bil-fire8/world-shared/global/metplus.data/obdata"
+    export ccpa_24hr_arch_dir="/gpfs/f6/bil-fire8/world-shared/global/metplus.data/obdata/ccpa_accum24hr"
+    export METviewer_AWS_scripts_dir="/gpfs/f6/bil-fire8/world-shared/global/metplus.data/VRFY/METviewer_AWS"
```

Comment:
Suggest building these paths with `$ACCOUNT` and `$USER`, e.g.

`export HOMEDIR="/gpfs/f6/${ACCOUNT}/scratch/${USER}"`